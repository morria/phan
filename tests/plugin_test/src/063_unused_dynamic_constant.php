<?php
namespace foo;

class C {
    const BAR = \foo\dynamic1;
}
define(__NAMESPACE__ . '\dynamic1', 'value');
define(__NAMESPACE__ . '\dynamic2', 'value');
echo C::BAR . "\n";
