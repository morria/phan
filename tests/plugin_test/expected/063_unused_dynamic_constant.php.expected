src/063_unused_dynamic_constant.php:8 PhanUnreferencedConstant Possibly zero references to global constant \foo\dynamic2
